distribution/lib/Standard/Table/0.0.0-dev/src/Data/Group_By.enso

```diff
@@ -0,0 +1,25 @@
+from Standard.Base import all
+
+## Group By Key
+type Group_By_Key
+    type Group_By_Key values
+
+    ## See if two keys are equal
+    == : Group_By_Key->Boolean
+    == that =
+        if that.values.length != this.values.length then False else
+            0.up_to this.values.length . all i->(this.values.unsafe_at i == that.values.unsafe_at i)
```

Comment:
`unsafe_at` suggests that `values : Vector`.
But this implementation of `==` is exactly the same as `Vector.==`. So why override it?
I think we can just rely on the autogenerated one, as it will have the same semantics, or if you really want to explicitly write it, at least do:
```suggestion
    == : Group_By_Key->Boolean
    == that =
        that.values == this.values
```

Unless there is some reason that I'm not seeing as to why it should be implemented separately?